Libraries/Network/FormData.js

```diff
@@ -65,21 +65,24 @@ class FormData {
     return this._parts.map(([name, value]) => {
       var contentDisposition = 'form-data; name="' + name + '"';
       var headers: Headers = {'content-disposition': contentDisposition};
-      if (typeof value === 'string') {
-        return {string: value, headers, fieldName: name};
-      }
 
       // The body part is a "blob", which in React Native just means
       // an object with a `uri` attribute. Optionally, it can also
       // have a `name` and `type` attribute to specify filename and
       // content type (cf. web Blob interface.)
-      if (typeof value.name === 'string') {
-        headers['content-disposition'] += '; filename="' + value.name + '"';
-      }
-      if (typeof value.type === 'string') {
-        headers['content-type'] = value.type;
+      if (typeof value === 'object' && value.uri) {
```

Comment:
Can you remove the `&& value.uri` to correct the case the @kmagiera mentions? It's not a big deal, but since we don't really support stringifying arbitrary objects then I don't see much benefit to this extra check.